docs/modules/ROOT/pages/explanations/decisions/mariadb.adoc

```diff
@@ -44,30 +47,24 @@ Some additional Notes:
 * Percona has a MySQL Operator: https://docs.percona.com/percona-operator-for-mysql/pxc/
 * MOCO Operator (MySQL Only): https://github.com/cybozu-go/moco
 ** Has a similar concept to handling clustering like the Patroni framework that is used by Stackgres.
+* New operator, independent from MariaDB project, MIT licensed:  https://github.com/mariadb-operator/mariadb-operator[New Mariadb Operator]
 
 == Decision
 
-https://github.com/bitnami/charts/tree/master/bitnami/mariadb-galera[Galera Bitnami Helm Chart].
+https://github.com/mariadb-operator/mariadb-operator[New Mariadb Operator].
 
-The instantiation of the Helm Charts will be handled by `+provider-kubernetes+`.
+New operator is currently the best what github offers, comes with variety of installation methods and is actively maintained.
 
 Advantages::
 
-* Less complexity than operators, but provides more features
-* Have been used in SPKS for years now
-* Supports Standalone and Galera
+* Supports almost everything we need
+* Easy to integrate with Appcat
+* Actively maintained
```

Comment:
As of the time I looked at the GitHub repo it had 387 commits, which isn't a lot. How can we state that it is "actively maintained"? What is the measurement we can actually state that?